infra/k8s/kops-weave/weave.yml

```diff
@@ -163,6 +163,16 @@ items:
           labels:
             name: weave-net
         spec:
+          initContainers:
+          - name: iproute-add
+            image: busybox
+            securityContext:
+              privileged: true
```

Comment:
We shouldn't need this, right?

Reply:
I am not sure to be honest, I think I added it on purpose. Will double-check.

Ultimately I am not happy with this block being here at all, as it has nothing to do with `Weave`, but not sure where to place it. Ideally the issue on coreos/flannel will get some love, and we will understand better why this route is not there in the first place.

Reply:
I've added it as a todo in the follow up (if I have time tomorrow) - we are just running `ip route` once, and then this process exits - I don't see any problem with not being specific with the `securityContext` and just giving root priveleges.